scripts/runtests.sh

```diff
@@ -254,6 +254,28 @@ python3 scripts/check-yaml-fields.py "$tmpdir"/var/log/installer/autoinstall-use
         'autoinstall.source.id="ubuntu-server-minimal"'
 grep -q 'finish: subiquity/Install/install/postinstall/run_unattended_upgrades: SUCCESS: downloading and installing security updates' $tmpdir/subiquity-server-debug.log
 
+clean
+LANG=C.UTF-8 timeout --foreground 60 \
+    python3 -m subiquity.cmd.tui \
+    --dry-run \
+    --output-base "$tmpdir" \
+    --machine-config examples/simple.json \
+    --autoinstall examples/autoinstall-ad.yaml \
+    --kernel-cmdline autoinstall \
+    --source-catalog examples/mixed-sources.yaml &
+sub_pid=$!
+until [[ -S "$tmpdir/socket" || $(ps -p "$sub_pid" > /dev/null ) == 1 ]]
+do
+    sleep 1
+done
+curl --unix-socket "$tmpdir/socket" -X POST 'http://localhost/meta/client_variant?variant=%22desktop%22'
+ad_info=$(curl --unix-socket "$tmpdir/socket" -X GET 'http://localhost/active_directory' | jq '.password="PassWord"')
```

Comment:
passw0rd is our standard canary value for the value we don't want to find in a file